### DISCUSSION TITLE
布局

### DISCUSSION THREAD

**Discussion by Charles-Lou** [electerm/electerm#3521] [Ideas]
可以在布局上让sftp 和 ssh 在同一个界面左右布局显示，以便于监视文件夹及文件的变化

**Comment by zxdong262** [comment]
将在后续版本加入

**Reply by Charles-Lou** [reply]
> 将在后续版本加入

期待了

**Reply by Jachy7788** [reply]
> 将在后续版本加入

可以让sftp只显示远程端吗，不显示本地端，或者本地/远程端的显示可编辑
另外我看目前sftp和ssh显示位置是自动调整的，可以加入手动调整固定功能吗

**Reply by zxdong262** [reply]
https://github.com/electerm/electerm/wiki/Custom-CSS-examples

**Reply by Jachy7788** [reply]
> https://github.com/electerm/electerm/wiki/Custom-CSS-examples

看到了，多谢

**Comment by zxdong262** [comment]
当前版本已经支持sftp和终端都显示
